src/ts/components/listselector.ts

```diff
@@ -77,12 +77,7 @@ export abstract class ListSelector<Config extends ListSelectorConfig> extends Co
   }
 
   private getItemIndex(key: string): number {
-    for (let index in this.items) {
-      if (key === this.items[index].key) {
-        return parseInt(index);
-      }
-    }
-    return -1;
+    return this.items.findIndex((item) => item.key === key);
```

Comment:
Just to be sure: Is this supported on old TVs?

Reply:
I thought so, since we already use it a couple of times. But according to MDN it's only available since Chrome 45 😳 Let me run some tests.

Reply:
I rewrote the function to not use `findIndex` :)